### ISSUE TITLE
Add primary and secondary colors + for light and dark variants and apply to UI components 

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/geniza#391]
- Please review [the colors here ](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=2%3A610)

![Screen Shot 2021-12-04 at 12.02.12 AM.png](https://images.zenhubusercontent.com/5bf307a69d595b459ec91bb1/79facaa3-04f5-4d35-824f-106502998862)

- I have applied it to the UI for light and dark variants so you can see how they are being used – I have used the darker reds for dark mode however I'm not sure it really works – I prefer the dark grey version more.

**Comment by richmanrachel** [comment]
I agree with gray for dark mode. 

I like the red in the light mode designs. Where will the other colors be used, @gissoo?

**Comment by rlskoeser** [comment]
I like the red. I think I prefer the red active to green active I'm a little unsure about using red and green together without other colors (muted Christmas?), but defer to you and Marina.

Will the keywords in context still be highlighted in blue, or have you just not updated that? What will the highlight color be in dark mode?

**Comment by blms** [comment]
- I agree that gray for dark mode is preferable.
- Agree with Rebecca that red/green reads Christmas!
- I think the red in dark mode could use slightly more saturation and slightly less brightness, but I don't feel strongly about this.
- Maybe it would make sense for normally darkened background sections in light mode (e.g. pagination) to appear lightened in dark mode? That seems to be the typical approach in dark modes, and it works better for my eyes.

Also, not sure if you came across this but it seems useful for thinking through dark themes: https://material.io/design/color/dark-theme.html#properties (They even have some examples for the lightened background sections for "elevation," and adapting reds!)

**Comment by gissoo** [comment]
@richmanrachel I'm trying to incorporate the colors in the logo, more than the overall site – I think in my conversations with Marina she really liked the red and while trying out the colors on the interface the red looks the most appropriate compared to the green and the blue – when trying to keep things minimal they don't all work well together as part of the visual design of the site  – 

@rlskoeser @blms regarding the Christmas look, I think it's very contextual – these are some of the images that inspired these colors <img width="1101" alt="Screen Shot 2021-12-07 at 9 58 54 AM" src="https://user-images.githubusercontent.com/44615389/145052762-96c6ec3f-2583-4bc2-8a23-fb464c9486dd.png">


- I can however make the red more muted (It was muted before but it looked very brown in the designs, it was hard to see it was actually red) 
- Another reason for it feeling like Christmas to you is probably because we are not using any warm tones in the designs which can wipe out the christmas feel – like the off-white / cream colors you see in the images above because they are overused in these types of projects and Marina also really disliked that tone. 
- we could make the link styles red too, I chose the green for it because it helps set links apart from the rest of the elements, it's brighter than the red so it makes it easier to recognize the links.

**Comment by gissoo** [comment]
@mrustow @richmanrachel let me know what you think about [these colors](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=314%3A21701). I added document images to the designs so we can see the colors better. – I came up with a slightly different green (v3). I do like the green versions more too, I know Marina loves the green which is important.

**Comment by mrustow** [comment]
OK, I've thought long and hard, and here's what I think.

1. For light mode, I much prefer the green V1. (The green V2 reads as grey in some contexts and too military in others.) The berry red is a great color, but in this context, red is too dominant. It distracts from the content. With the green, the focus is on the manuscript images.
2. But for dark mode, I much prefer the pink. Dark mode in green is too dark. Can we do green for light mode and pink for dark mode?
3. The tapered stroke separating the search results: I would prefer to have it go in a single direction. The reversal of direction seems like it should be informational, yet it's not, and it's a bit visually confusing.
4. The aleph before the pagination of search results: I love it and would be sad to lose it, but same issue here: it seems like it's supposed to be informational, yet it's not. I like the way it anchors and balances the page. But it's an eye magnet, and then it's frustrating that it's not functional.
5. I love the banner up top with the handwritten text! Works perfectly.
6. I feel like there's some color missing from the top section of the page, between the banner and the search result pagination. Could there be a color application to the cluster-view toggle and/or the sort-by dropdown menu?
7. I'm having second thoughts about the squiggle as the top-menu highlight. As much as I adore it, it seems to be coming from a different design vocabulary from the rest of the site. I wonder what it would look like if we replaced it with a scaled-down version of the ombré line we're using as a results separator?
8. If we go with green for light mode, I would find it a bit disturbing that the logo has aquamarine and teal and the site has green. What if we asked Laurel to do a version of the logo replacing the teal (#00A19B) with green?

**Comment by gissoo** [comment]
> OK, I've thought long and hard, and here's what I think.
> 
>     1. For light mode, I much prefer the green V1. (The green V2 reads as grey in some contexts and too military in others.) The berry red is a great color, but in this context, red is too dominant. It distracts from the content. With the green, the focus is on the manuscript images.
> 

@mrustow:
- this makes sense! I agree. 


>     2. But for dark mode, I much prefer the pink. Dark mode in green is too dark. Can we do green for light mode and pink for dark mode?
> 

- I don't have any issues especially if we are incorporating both of those colors with the logo – but I wonder if @rlskoeser and @blms have any concerns on their side with implementation logic. 


>     3. The tapered stroke separating the search results: I would prefer to have it go in a single direction. The reversal of direction seems like it should be informational, yet it's not, and it's a bit visually confusing.
> 

- [here it is](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=324%3A13660), how do you feel about it? 

>     4. The aleph before the pagination of search results: I love it and would be sad to lose it, but same issue here: it seems like it's supposed to be informational, yet it's not. I like the way it anchors and balances the page. But it's an eye magnet, and then it's frustrating that it's not functional.
> 

- what about [the revised version, here](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=324%3A13660)? Or do you prefer to omit it entirely? @rlskoeser @blms would the alef work how I have it on the page, or should we just omit it?

>     5. I love the banner up top with the handwritten text! Works perfectly.
> 

- great! 

>     6. I feel like there's some color missing from the top section of the page, between the banner and the search result pagination. Could there be a color application to the cluster-view toggle and/or the sort-by dropdown menu?
> 

- Marina, it's hard to really tell how the color is being incorporated since these pages are static, however I have placed the search components separately [here](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=340%3A18860) for you to see – on clickable components the green is intended to be used once there is an interaction(click or hover), because that's when a high contrast in the color of the component needs to appear as a response to the user – what about the revised versions with [green gradient on top portion of the page](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=338%3A18015) or [the entire page](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=338%3A14199)?

>     7. I'm having second thoughts about the squiggle as the top-menu highlight. As much as I adore it, it seems to be coming from a different design vocabulary from the rest of the site. I wonder what it would look like if we replaced it with a scaled-down version of the ombré line we're using as a results separator?
> 
 
- how do you feel now? [here you can see the revised version](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=324%3A13660). It does match with the UI more, however I drew the squiggle because you had mentioned you preferred the soft angles for the logo so wanted to make sure we include more elements with the same softness on the site, but I think we are already incorporating that on the homepage. 

>     8. If we go with green for light mode, I would find it a bit disturbing that the logo has aquamarine and teal and the site has green. What if we asked Laurel to do a version of the logo replacing the teal (#00A19B) with green?

- The Geniza lab logo, correct? When I previously asked you about the concept behind the lab's logo you informed me that its design process was very much influenced by textiles, I don't recall whether the designer had referred to the Geniza fragment images specifically since the Geniza lab does not only work on PGP. I haven't seen the teal be used anywhere on the fragments. PGP's role in the lab's establishment would be a reason to apply the green and omit the teal, unless you have other thoughts after reading my questions below.

- Think about these questions: Is there any significance to the teal color that you picked for the logo and how it's used on the lab's site? Does that color help represent something beyond PGP that could be helpful to the lab's work in the future? Do you have plans in the short term to shift focus on projects other than PGP that would be equally important as PGP but where the green we picked will not be relevant? 

Just note, that if they change the teal to green anything that uses that color on the site will need to become green which could work everywhere except on the blue they have used in the footer (for accessibility reasons). (on the lab's site the teal used in the header, footer, the middle container, the buttons) – the sites would be visually somewhat close to each other, not sure if that is what you want.

**Comment by blms** [comment]
For different colors in light vs dark mode, it's no different on the implementation end because it will be two different colors either way.

Regarding the aleph, I prefer it without—and of course it's easier to implement without! But doable if we want it.

**Comment by rlskoeser** [comment]
I agree with Ben — can't think of an implementation concern with the dark mode colors being different, since they're specified separately in the style sheet.  I don't think the dark mode and light mode need necessarily be based on the same color scheme.

I agree with Marina that the aleph isn't really meaningful. Let's leave it off for the MVP, at least — if you think of a way to incorporate it later on that is more meaningful we could add it.

FWIW, I like the green gradient (the middle option, where there's just a slight gradient at the top) — it makes the page feel much less stark than the page without any gradient, and it works nicely with the search form inputs.

**Comment by mrustow** [comment]
- tapered stroke: I like it better when they all move in the same direction. I have a feeling the page will look more balanced if they move in the opposite direction, though. 
- the aleph (borges reference!): can we try a handwritten aleph taken from a manuscript? it could read more immediately as decorative rather than functional. but fine to pause this until after MVP.
- clickable components: ah ok, makes sense, thanks. i think that will address my concern about the top part of the page, in addition to reversing the tapered stroke. i'm not a fan of the gradient background; it makes the green feel too pervasive. i could go with a partial gradient like the middle one in theory, but i would want a different color (or grey). but i think it's good without.  
- the squiggle: actually i miss it :( suggestions: keep the squiggle, but more tapered at both ends and more gradient on the left in addition to the right; or lose the squiggle and keep the tapered line, but give it a longer tail (as below) and more of a gradient?
- the logo: oh dang, i forgot about the teal on the PGL site. you're right. it's too complicated to change it, and the concern i expressed is not a big one, so let's drop it. 
- sorry it took me so long to respond. i've been back to back in meetings (3x) and seminars (3x!) all day.

**Comment by gissoo** [comment]
Have added finalized colors:

Done and approved:

- [for light mode](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=435%3A29939)

- [for dark mode](https://www.figma.com/file/anZFStv7zBaVWnFxr95nKt/Work-in-Progress-%2F-WIP?node-id=435%3A29949)

**Comment by gissoo** [comment]
Documenting convos with Marina on Slack:

Marina: 

1. did you ever try reversing the tapered lines that divide the records, so the thicker part is on the left? what’s your reasoning for putting the thicker part on the right?

2. The top menu on the upper right: the kerning is bothering me. It looks good in “search,” but it’s weirdly gappy in Editorial, Contact Us and About.

3. I’m not crazy about how the font looks in the logo in the word “Geniza.” Is it the same as the one you’re tweaking and it just looks different naked? Who knows, anyway like I say it’s totally irrelevant for MVP, which is just a tool for us to keep thinking through the site!

For the rest, it looks fantastic. I love the top and bottom banners, I love the nice, balanced design, the green looks great (I haven’t looked at dark mode yet, stay tuned).


Gissoo: 

1. I did – the reason I kept them on the right is because I want to highlight the rtl nature of this project – this type of thickness always begins from the right – a. after revising from the alternating version my intentions was to always keep it on the right – b. but if we want we can do it from left to right in the latin version and right to left in the Hebrew and Arabic versions – let me know – I think there is something novel about respecting the original orientation regardless of what language the site is in – but let me know if you disagree and we can pursue the option “b”

2. your eyes are gem, Marina!! (knock on wood) – there was an unintended 1px kerning there that’s why they were gappy, there is no kerning in the type styles I have submitted for implementation so they have the correct versions, and this here was just something that slipped from my eyes! fixed it now

3. It’s the same font, yes, let’s come back to it in Jan, it’s getting closer
